R/functions_degs.R

```diff
@@ -1,17 +1,1391 @@
+#' Sets up a list with contrasts for analysis. Makes sure that all required information is available. Sets default and does basic checks. 
```

Comment:
I did not check this in detail. I see a lot of new code. Can you please make sure to remove the old deprecated code from scrnaseq version 1?